static/js/components/HomePage.jsx

```diff
@@ -12,43 +12,50 @@ import SourceList from "./SourceList";
 import GroupList from "./GroupList";
 import NewsFeed from "./NewsFeed";
 import TopSources from "./TopSources";
+import SourceCounts from "./SourceCounts";
+
 import styles from "./HomePage.css";
 
 const ResponsiveGridLayout = WidthProvider(Responsive);
 
 const xlgLayout = [
-  { i: "sourceList", x: 0, y: 0, w: 9, h: 6, minW: 9, isResizable: false },
-  { i: "newsFeed", x: 9, y: 0, w: 7, h: 3, isResizable: false },
-  { i: "topSources", x: 9, y: 0, w: 4, h: 2, isResizable: false },
-  { i: "groups", x: 13, y: 0, w: 2, h: 2, isResizable: false },
+  { i: "sourceCounts", x: 0, y: 0, w: 2, h: 2, isResizable: false },
```

Comment:
Can we leave this for K to address later? Bound to cause conflicts, and not sure we want to hold up this PR for any tuning here.

Reply:
Stefan has a good point - we'll be changing these a lot as all the new widgets we have planned for this iteration come in. Might be better to just do a separate PR at the end to reorganize all the widgets at once after they get in. We can maybe just change these locally to test the new widgets, but not include any layout changes in the PRs until we have a more stable set of home page widgets in.

Reply:
Good idea. Let's leave in these changes and then in future PRs hold off